[2022-01-25 06:36:00,122] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-19T00:00:00+00:00 [queued]>
[2022-01-25 06:36:00,133] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-19T00:00:00+00:00 [queued]>
[2022-01-25 06:36:00,133] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:36:00,133] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 06:36:00,133] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 06:36:00,149] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-19 00:00:00+00:00
[2022-01-25 06:36:00,153] {standard_task_runner.py:52} INFO - Started process 3892 to run task
[2022-01-25 06:36:00,157] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-19T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpm_q4z_uv', '--error-file', '/tmp/tmpncbb8tte']
[2022-01-25 06:36:00,162] {standard_task_runner.py:77} INFO - Job 9: Subtask first_task
[2022-01-25 06:36:00,242] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-19T00:00:00+00:00 [running]> on host ceab856ace11
[2022-01-25 06:36:00,402] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-19T00:00:00+00:00
[2022-01-25 06:36:00,402] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 06:36:00,403] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 06:36:00,412] {subprocess.py:85} INFO - Output:
[2022-01-25 06:36:00,414] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 06:36:00,415] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 06:36:00,439] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220119T000000, start_date=20220125T063600, end_date=20220125T063600
[2022-01-25 06:36:00,490] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 06:36:00,519] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-01-25 07:27:43,325] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-19T00:00:00+00:00 [queued]>
[2022-01-25 07:27:43,337] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-19T00:00:00+00:00 [queued]>
[2022-01-25 07:27:43,337] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:43,337] {taskinstance.py:1239} INFO - Starting attempt 1 of 6
[2022-01-25 07:27:43,337] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 07:27:43,347] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): first_task> on 2022-01-19 00:00:00+00:00
[2022-01-25 07:27:43,352] {standard_task_runner.py:52} INFO - Started process 353 to run task
[2022-01-25 07:27:43,356] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'this_is_first_dag', 'first_task', 'scheduled__2022-01-19T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpmfs8pb_u', '--error-file', '/tmp/tmpx_qs_1x1']
[2022-01-25 07:27:43,360] {standard_task_runner.py:77} INFO - Job 9: Subtask first_task
[2022-01-25 07:27:43,529] {logging_mixin.py:109} INFO - Running <TaskInstance: this_is_first_dag.first_task scheduled__2022-01-19T00:00:00+00:00 [running]> on host 943a4c094194
[2022-01-25 07:27:43,580] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflaw_test
AIRFLOW_CTX_DAG_ID=this_is_first_dag
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-01-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-19T00:00:00+00:00
[2022-01-25 07:27:43,581] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-01-25 07:27:43,581] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo Hello, this is my first dag. Are you OK?']
[2022-01-25 07:27:43,590] {subprocess.py:85} INFO - Output:
[2022-01-25 07:27:43,592] {subprocess.py:89} INFO - Hello, this is my first dag. Are you OK?
[2022-01-25 07:27:43,592] {subprocess.py:93} INFO - Command exited with return code 0
[2022-01-25 07:27:43,616] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=this_is_first_dag, task_id=first_task, execution_date=20220119T000000, start_date=20220125T072743, end_date=20220125T072743
[2022-01-25 07:27:43,648] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-01-25 07:27:43,677] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
